enterprise/login-email-password-setup.md

```diff
@@ -0,0 +1,7 @@
+# Login with email and password
+## Private Packagist Enterprise
+
+## Setup
+Navigate to the "Global Configuration" section from the admin page to get to the form below. You can enable login and account creation with password by selecting the option in the form and submitting the form via the "Update configuration" button.
```

Comment:
with _a_ password. But better "Enable login and account registration with email addresses and passwords by .."